corehq/apps/accounting/user_text.py

```diff
@@ -273,8 +267,13 @@ def get_columns(cls, feature):
             cls.ANDROID_GATEWAY: (False, True, True, True, True),
             cls.SMS_DATA_COLLECTION: (False, False, True, True, True),
             cls.INBOUND_SMS: (False, False, True, True, True),
-            cls.INCLUDED_SMS_DIMAGI: (False, _("100 /month"), _("500 /month"), _("1,000 /month"), _("2,000 /month")),
-            cls.INCLUDED_SMS_CUSTOM: (False, _("1 cent/SMS"), _("1 cent/SMS"), _("1 cent/SMS"), _("1 cent/SMS")),
+            cls.SMS_PRICING: (
+                False, 
+                ugettext_noop(mark_safe(_('<a target="_blank" href="%(url)s">Click Here</a>.') % {'url': (reverse(PublicSMSRatesView.urlname)),},)), 
```

Comment:
shouldn't "Click Here" be the only text that's translated. Also, line is very long.